.drone.yml

```diff
@@ -1,124 +1,151 @@
-pipeline:
-  build_10:
-    image: node:10
-    commands:
-      - npm --loglevel warn install
-      - npm test
-    when:
-      event: [push, pull_request]
+---
+kind: pipeline
+name: default
+type: kubernetes
 
-  build_brpapp:
-    image: docker:18.03.1-ce
-    environment:
-      - DOCKER_HOST=tcp://172.17.0.1:2375
-    commands:
-      - docker build -t app-$${DRONE_COMMIT_SHA} .
-    when:
-      event: pull_request
+steps:
+- name: build_10
```

Comment:
Can we name these pipeline steps similarly in all of the pipelines

Reply:
changed the step to build, but it is not present in ARE